main.tf

```diff
@@ -71,6 +71,25 @@ resource "ibm_iam_authorization_policy" "scc_wp_s2s_access" {
 data "ibm_iam_account_settings" "iam_account_settings" {
 }
 
+resource "ibm_scc_instance_settings" "scc_instance_settings" {
```

Comment:
can you leave a code comment above this resource block to explain what its doing (aka attaching a COS bucket and an event notifications instance)